index.js

```diff
@@ -7,6 +7,6 @@ const batchSize = 500;
 const batchTime = 5000;
 
 const serverLogger = new ServerLogger(batchSize, batchTime);
-export function createClient(appId) {
-	return new LoggingClient(appId, serverLogger);
+export function createClient(appId, shouldThrottle) {
```

Comment:
Might suggest doing this as an `opts` object so we can add more options later without needing to expand the list of parameters and force people to provide options they don't want to.

Reply:
Better do this now than later, don't forget to bump the major if you do this.